packages/extension-browser/src/devtools/views/pages/config.tsx

```diff
@@ -52,33 +52,41 @@ const ConfigPage = ({ onStart }: Props) => {
 
         const ignoredUrls = await resolveIgnoreQuery(config.ignoredUrls);
 
-        onStart({...config, ignoredUrls});
+        onStart({ ...config, ignoredUrls });
     }, [config, onStart]);
 
     const onCategoriesChange = useCallback((disabledCategories?: string[]) => {
-        setConfig({...config, disabledCategories});
+        setConfig({ ...config, disabledCategories });
     }, [config]);
 
     const onBrowsersChange = useCallback((browserslist?: string) => {
-        setConfig({...config, browserslist});
+        setConfig({ ...config, browserslist });
     }, [config]);
 
     const onResourcesChange = useCallback((ignoredUrls?: string) => {
-        setConfig({...config, ignoredUrls});
+        setConfig({ ...config, ignoredUrls });
     }, [config]);
 
     const onRestoreClick = useCallback(() => {
         setConfig({});
     }, []);
 
     return (
-        <Page title={getMessage('configurationTitle')} className={styles.root} actionName={getMessage('analyzeButtonLabel')} onAction={onAnalyzeClick}>
+        <Page title={getMessage('configurationTitle')} className={styles.root} actionName={getMessage('startScanButtonLabel')} onAction={onAnalyzeClick}>
+            <div className={styles.categories}>
+                <CategoriesConfig disabled={config.disabledCategories} onChange={onCategoriesChange} />
+                <BrowsersConfig query={config.browserslist} onChange={onBrowsersChange} />
+                <ResourcesConfig query={config.ignoredUrls} onChange={onResourcesChange} />
+            </div>
             <AppButton className={styles.restoreButton} onClick={onRestoreClick}>
                 {getMessage('restoreDefaultsLabel')}
             </AppButton>
-            <CategoriesConfig disabled={config.disabledCategories} onChange={onCategoriesChange}/>
-            <BrowsersConfig query={config.browserslist} onChange={onBrowsersChange}/>
-            <ResourcesConfig query={config.ignoredUrls} onChange={onResourcesChange}/>
+            {' '}
+            <div className={styles.poweredBy}>
+                {getMessage('poweredByLabel')}
+                {' '}
+                <a href="https://webhint.io" rel="noopener" target="_blank">webhint</a>
```

Comment:
I know it wasn't there where this was copied from, but we should include `noreferrer` as well.
```suggestion
                <a href="https://webhint.io" rel="noopener noreferrer" target="_blank">webhint</a>
```